rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/impl/UriBuilderImpl.java

```diff
@@ -661,6 +662,14 @@ private void setUriParts(URI uri) {
         }
     }
 
+    private boolean queryIsEmpty() {
+        return query == null || query.isEmpty();
```

Comment:
We have a helper method for it,  `CollectionUtils.isEmpty(query)` from `org.apache.cxf.common.util`